js/main.js

```diff
@@ -59,6 +62,44 @@ function createArtPieces(event) {
   $artList.appendChild($artAddDivRow);
 }
 
+function createSavedArtPieces(event) {
+
+  var newDivColumnHalf = document.createElement('div');
+  var newLi = document.createElement('li');
+  var newArtImg = document.createElement('img');
+  var newArtArtist = document.createElement('h2');
+  var newArtTitleYear = document.createElement('p');
+  var newItalics = document.createElement('i');
+  var addArtIcon = document.createElement('img');
+  var newArtDescription = document.createElement('p');
+
+  newDivColumnHalf.className = 'column-half';
+  newArtArtist.className = 'artist-name-header';
+  newArtTitleYear.className = 'art-name-year';
+  newArtDescription.className = 'art-description';
+
+  newArtImg.setAttribute('id', event.artId);
+  newArtImg.setAttribute('src', event.imageURL);
+  addArtIcon.setAttribute('src', 'images/minus-sign.svg');
+  addArtIcon.className = 'remove-icon';
+
+  newArtArtist.textContent = event.artistName;
+  newItalics.textContent = event.artTitle;
+
+  $savedArtAddDivRow.appendChild(newDivColumnHalf);
+  newDivColumnHalf.appendChild(newLi);
+  newArtArtist.appendChild(addArtIcon);
+  newLi.appendChild(newArtImg);
+  newLi.appendChild(newArtArtist);
+  newLi.appendChild(newArtTitleYear);
+  newLi.appendChild(newArtDescription);
+
+  var firstChild = newArtTitleYear.firstChild;
+  newArtTitleYear.insertBefore(newItalics, firstChild);
+
+  $savedArtList.appendChild($savedArtAddDivRow);
+}
```

Comment:
Instead of changing the DOM directly, you should pass in the data for the entry and `return` the dom tree for it, then append it where that function is called.